hack/ipa_debug_tools.sh

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bash
+
+# The path to the directory that holds this script
+CURRENT_SCRIPT_DIR="$(dirname -- "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+# This ceates a minimal set up in /tmp/debug-initramfs directory for
+# comparing built ipa initramfs the master ipa initramfs.
+set_up_debug_dirs() {
+    # Export DISABLE_UPLOAD, ENABLE_BOOTSTRAP_TEST, TEST_IN_CI to allow local
+    # test run.
+    export DISABLE_UPLOAD="true"
+    export ENABLE_BOOTSTRAP_TEST="false"
+    export TEST_IN_CI="false"
+
+    # Build ipa with build_ipa.sh
+    cd "$CURRENT_SCRIPT_DIR/../jenkins/scripts/dynamic_worker_workflow" || exit
```

Comment:
```suggestion
    cd "${CURRENT_SCRIPT_DIR}/../jenkins/scripts/dynamic_worker_workflow"
```
`|| exit 1` not needed if we're running `set -e` mentioned earlier, cleans up the code.